### ISSUE TITLE
Cantarell is *still* out of date.

### ISSUE THREAD

**Issue by SushiDude** [google/fonts#112]
See [my old report](https://code.google.com/archive/p/googlefontdirectory/issues/290) that was made **over one and a half years ago**. Be sure to read comment number two.

The proper location to retrieve the font is: https://download.gnome.org/sources/cantarell-fonts/0.0/

The font is developed here: https://git.gnome.org/browse/cantarell-fonts

The latest version is 0.0.16.

**Comment by fitojb** [comment]
IIRC, the reason why Dave's original version of Cantarell was not updated in Google Fonts was because the Gnome-developed branch removed italics, as stated by him. And the differences in upright versus italics are significant enough to make them look really weird side-by-side.

**Comment by adrientetar** [comment]
Weren't current italics autogenerated?

**Comment by davelab6** [comment]
I used FontForge's tools to derive the italics from the roman, but the process took about 40 hours.

**Comment by davelab6** [comment]
@SushiDude https://www.google.com/fonts/specimen/Cambay may work for you; its further developed than the GNOME Cantarell.

**Comment by davelab6** [comment]
> the differences in upright versus italics are significant enough to make them look really weird side-by-side

Maybe GNOME should switch to Camby

**Comment by fitojb** [comment]
@davelab6 You’ve misinterpreted what I said: the differences from upright and italics I mentioned last year were in relation to Google’s copy and GNOME’s. That comment was to explain why was Google Fonts still hosting the typeface’s first version.

Besides, the development of Cambay has been merged back in GNOME’s Cantarell, so it wouldn’t be anything to gain for GNOME to switch to Cambay. Also, its metrics are different due to its addition of Devanagari.

**Comment by SushiDude** [comment]
@fitojb [Since version 0.0.19, Cantarell includes a generated oblique version.](https://git.gnome.org/browse/cantarell-fonts/tree/NEWS?h=0.0.24#n35)

**Comment by fitojb** [comment]
@SushiDude Yes, I know =)

**Comment by laerm0** [comment]
I see the Gnome Cantarell was updated a little over a year ago to 0.0.25. Is their development path good for us and should we update with their data?

**Comment by madig** [comment]
No you shouldn't :) I'm working on a re-design: https://github.com/madig/cantarell-fonts.

Dave told me the other forks diverge too far from his design intentions, which is why none of them took the official GF spot yet. I'm working on making it there :)

**Comment by madig** [comment]
@m4rc1e Have a look at https://github.com/madig/cantarell-fonts

**Comment by davelab6** [comment]
Happy to say that @madig has been hard at work on this and is ready for @m4rc1e to prep a PR for it. 

@madig please confirm where marc should pull from?

**Comment by m4rc1e** [comment]
@madig Thanks, I'll get QAing.

**Comment by m4rc1e** [comment]
Alright, just had a quick look.

Firstly, I love the chain. It builds nicely and does what it says it will do.

We need Italics before I can look further into this. When we upgrade fonts at GF, we must replace all the existing styles.

@davelab6 Does Cantarell have faux italics?

If so, we could use a customParam to do this. We may also have to implement this param in glyphsLib.

**Comment by madig** [comment]
Oh boy, italics. I'd need param-faux-italics to work outside Glyphs. Not sure if glyphsLib is up to the task?

**Comment by m4rc1e** [comment]
@madig yeah, its quite hardcore. An alternative would be to use fontTool's misc transform module. Roboto uses this approach.

https://github.com/fonttools/fonttools/blob/master/Lib/fontTools/misc/transform.py

**Comment by madig** [comment]
This is very, very dirty but I'll have a look when I get home 😁

**Comment by emmamarichal** [comment]
Hello everyone and nice to meet you @madig! I'm Emma, I work with the GF team. 
I wanted to know how the project is going? I looked at the repo and saw that some commits are from 2022, but I don't see any italics. What's the status of the project? :)

**Comment by madig** [comment]
The status is that I still have no energy to either draw italics or skew them :) I'd be glad to hand that off to someone else.

**Comment by qwertychouskie** [comment]
Just wanted to check in on the status of this.  The version currently shipped by Google Fonts causes some websites to display very wrongly, e.g. https://extensions-next.gnome.org/:

Bad version from Google Fonts:
<img width="289" alt="image" src="https://github.com/user-attachments/assets/bbaf147a-a553-4ebd-a50e-fec18603758c">

Good version from Gnome repository:
<img width="289" alt="image" src="https://github.com/user-attachments/assets/80cd865e-5bc6-4d88-80e4-8abfc4313945">

I know the policy normally would be to wait for upstream to provide a dedicated italic version, but the very old version of the font currently shipped by Google Fonts is breaking real sites, so the theoretical advantage of not removing italic support is likely being outweighed by the real-world breakage the old font is causing.

**Comment by madig** [comment]
The status is: I asked my employer if I could do anything to push Cantarell forward and they asked me not to, because they are also in the font development business and don't want me to put out a free competitor :) So, unless someone commissions my employer, I can't help with a proper Italic.
